src/cpr_data_access/yql_builder.py

```diff
@@ -121,17 +121,15 @@ def build_document_filter(self) -> Optional[str]:
             return f"(document_import_id in({documents}))"
         return None
 
-    def build_keyword_filter(self) -> Optional[str]:
-        """Create the part of the query that adds keyword filters"""
-        keyword_filters = self.params.keyword_filters
-        if keyword_filters:
-            filters = []
-            for field_name, values in keyword_filters.items():
-                for value in values:
-                    filters.append(f'({field_name} contains "{sanitize(value)}")')
-            if filters:
-                return f"({' or '.join(filters)})"
-        return None
+    def _inclusive_keyword_filters(
+        self, keyword_filters: KeywordFilters, field_name: str
+    ):
+        values = getattr(keyword_filters, field_name)
+        filters = []
+        for value in values:
+            filters.append(f'({field_name} contains "{sanitize(value)}")')
+        if filters:
+            return f"({' or '.join(filters)})"
```

Comment:
Is this not still and `or` statement as opposed to an `and`?

Reply:
Its an OR within each filter:
```
canada OR usa
```

but across filters its an and, see the `build_where_clause` method:
```
(canada OR usa) AND (legistaltive)
```

Reply:
Oh yes I see it on line 163, cool!